python/src/uagents/context.py

```diff
@@ -374,6 +386,10 @@ async def send_raw(
         Returns:
             MsgStatus: The delivery status of the message.
         """
+
+        # Destination without ledger prefix
+        raw_destination = destination[-65:]
```

Comment:
Things get a bit complicated in here because destination can also be a name at this point. I guess it's what we're calling an `identifier` now, and will need a way to parse this. The resolver should handle most of it, but we might need a simple function to extract the agent address from an identifier for things like checking if it is registered with the local dispatcher.

Reply:
@jrriehl I updated the code with a proposal for this, let me know what you think